.github/workflows/build-release.yml

```
@@ -189,3 +189,46 @@ jobs:
 
           cd ../installation
           ./bin/python3 -m test -uall
+
+  build-android:
+    needs:
+      - verify-input
+
+    # Android binary releases began in Python 3.14.
+    if: |
+       !(
+          startsWith(needs.verify-input.outputs.cpython_release, '3.9.') ||
+          startsWith(needs.verify-input.outputs.cpython_release, '3.10.') ||
+          startsWith(needs.verify-input.outputs.cpython_release, '3.11.') ||
+          startsWith(needs.verify-input.outputs.cpython_release, '3.12.') ||
+          startsWith(needs.verify-input.outputs.cpython_release, '3.13.')
+       )
```

I'd suggest computing another boolean output within `verify-input` similar to how `build-context` functions in `python/cpython`, which would help consolidate the decision logic closer to other checks.

And the use would look as follows:
```suggestion
    if: fromJSON(needs.verify-input.outputs.build-android)
```